indico/modules/events/registration/client/js/form/fields/ChoicesSetup.jsx

```diff
@@ -138,6 +140,11 @@ Choices.defaultProps = {
   forAccommodation: false,
 };
 
+export const choiceFieldsSettingsFormDecorator = createDecorator({
+  field: 'choices',
+  updates: choices => ({isPriceSet: choices.some(c => c.price)}),
```

Comment:
Doesn't this mean that we end up sending `isPriceSet` in the form data when submitting now?

Reply:
OK this doesn't seem to happen because `getValuesForFields` in `onSubmit` only uses values for registered fields.

Reply:
Good. But still, is there a cleaner way to implement this functionality? Or should I leave it as is?

Reply:
It's certainly a creative solution... for now I'd leave it as it is.